oauth2-provider.gemspec

```diff
@@ -17,6 +17,8 @@ spec = Gem::Specification.new do |s|
   s.add_dependency 'json'
   s.add_dependency 'rack'
   s.add_dependency 'json-jwt'
+  s.add_dependency 'jwt'
+  s.add_dependency 'flick_auth'
```

Comment:
Need to remove this dependency, I couldn't see anything that uses Flick Auth? Just the JWT stuff directly.